Lib/test/test_traceback.py

```diff
@@ -387,7 +396,7 @@ def test_encoded_file(self):
         process = subprocess.Popen([sys.executable, "-c",
                                     "import sys; print(sys.stdout.encoding)"],
                                    stdout=subprocess.PIPE,
-                                   stderr=subprocess.STDOUT)
+                                   stderr=subprocess.STDOUT, env={})
```

Comment:
On some platforms I believe things can fail to execute at all with an empty environment missing some required information...?   should these tests use `test_support.script_helper.run_python_until_end()`?

Reply:
Now that I am deactivating on `-E` this change is not needed anymore.